pkg/policy/nsaHardeningRules.yaml

```diff
@@ -0,0 +1,359 @@
+- id: 35
+    name: Ensure containers have a read-only root filesystem
+    uniqueName: NSA_INCORRECT_READONLYROOTFILESYSTEM_VALUE
+    enabledByDefault: true
+    documentationUrl: 'https://hub.datree.io/'
+    messageOnFailure: Incorrect value for key `readOnlyRootFilesystem` - set to 'true' 
+    category: NSA-Hardening
+    schema:
+      definitions:
+        containerSecurityPattern:
+          properties:
+            spec:
+              properties:
+                containers:
+                  type: array
+                  items:
+                    properties:
+                      securityContext:
+                        properties:
+                          readOnlyRootFilesystem:
+                            enum:
+                              - true
+                              - "true"
+                        required:
+                        - readOnlyRootFilesystem
+                    required:
+                    - securityContext
+        podSecurityContextPattern:
+          if:
+            properties:
+              kind:
+                enum:
+                  - Pod
+          then:
+            properties:
+              spec:
+                properties:
+                  securityContext:
+                    properties:
+                      readOnlyRootFilesystem:
+                        enum:
```

Comment:
is this a string or a boolean? if it's a boolean - use const:
https://stackoverflow.com/a/57146508/9595106